File: src/main/java/no/nav/veilarboppfolging/controller/v2/response/ManuellStatusV2Response.java
package no.nav.veilarboppfolging.controller.v2.response;


public class ManuellStatusV2Response {
    boolean erUnderManuellOppfolging;
    KrrStatus krrStatus;

    public static class KrrStatus {
        boolean kanVarsles;
        boolean erReservert;
    }
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/v2/response/ManuellStatusV2Response.kt
package no.nav.veilarboppfolging.controller.v2.response


data class ManuellStatusV2Response(
    val erUnderManuellOppfolging: Boolean,
    val krrStatus: KrrStatus,
) {
    data class KrrStatus(
        val kanVarsles: Boolean,
        val erReservert: Boolean,
    )
}
